### DISCUSSION TITLE
Respecto a la optimización del código

### DISCUSSION THREAD

**Discussion by Monti7475** [IIC2133-PUC/2024-1#42] [Tarea 0] [answered]
Habrán desvuentos si el código es ineficiente?
Es decir, cumple la tarea adecuadamente pero se da algunas vueltas que podrian ser omitidas

**Comment by benjavicente** [comment, marked as answer]
Hola! La tarea tiene que correr en un tiempo adecuado (0.1s a 1s, como máximo 10s) y memoria adecuada (100kb a 300kb es ideal en esta tarea, a lo más 10Gb pero prob sea porque hay algo malo). Ve eso con los test grandes (hard).

La tarea puede optimizar _mucho_, pero lo que se espera es que tengan una complejidad adecuada (ej: no tener sorting de O(n^3)).
